beaker_server/root/usr/bin/init.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+function log_info {
+  echo "---> `date +%T`     $@"
+}
+
+# Poll until MySQL responds to our ping.
+function wait_for_mysql() {
+  while ! mysqladmin -h 172.20.0.10 -u root -ppassword ping &>/dev/null; do
```

Comment:
Should we limit the number of attempts to connect in case there is a problem with the DB?  Maybe 2 minutes?  It seems the DB is usually ready in about 30 seconds.